.gitignore

```diff
@@ -24,6 +24,7 @@ build/
 bld/
 [Bb]in/
 [Oo]bj/
+localpkg/
```

Comment:
It's somewhat unrelated with this change. build.sh creates localpkg folder under src (for reasons that I didn't dig into). It does so even on the baseline. It's producing an annoying `git status`, and even more annoying when you try to do `git add -A`.